README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://github.com/pymc-devs/pymc3/blob/master/docs/pymc3_logo.jpg?raw=true
+.. image:: https://cloud.githubusercontent.com/assets/781659/26266864/0d462598-3c9c-11e7-8697-2e811e231694.png
```

Comment:
I see. I hope there is no problem to hold 2 or three logos.

1. Old one for legacy references
2. transparent that I've posted above
3. Just square rocket